.github/workflows/cd-dev.yml

```diff
@@ -3,16 +3,18 @@ name: cd-dev
 on:
   push:
     branches:
-      - master
-
+      - master  
+  pull_request: ## TODO:Remove after validating
+      branches:
+      - master  
```

Comment:
use workflow_dispatch please, I had already posted this comment elsewhere

Reply:
workflow_dispatch doesn't work from feature branch, just when configured in main